### DISCUSSION TITLE
Accuracy spectrum

### DISCUSSION THREAD

**Discussion by Frous-s** [ppy/osu#19512] [Ideas]
![acc spectrum](https://user-images.githubusercontent.com/66706431/182067514-92524f48-2697-4820-96e7-ba92c06fc027.png)
Pretty understandable with the image attached, make green and yellows a spectrum from 1-80% instead of current accuracy, leave blue unchanged as it will be too punishing for players

Remove current loss for pp on the first 100s of 50s being more pp loss than before, instead just make it be affected by total acc(just a multiplier) and misses and other factors like the type of pattern

leave the same treshhold for 50s and 100s in the scoring system to keep it consistent

**Comment by smoogipoo** [comment]
I can't understand what you're suggesting.

If you're suggesting to make accuracy depending on time delta, that's not going to happen.

**Reply by Frous-s** [reply]
I don't think so. Time delta does not sound like what I'm suggesting, I still have some doubts about exactly what you mean with that term, but it doesn't seem to fit with the concept I'm explaining here.

The suggestion is basically removing green and yellows being set values of 33.33% and 16.66% and instead be a wide range of values(maybe of 1% precision) based on timing the section of the tap.

Blues should be unchanged as they would be too unfair for overall acc.

**Reply by ghost** [reply]
Time delta refers to the amount of time between two specific timestamps.

I feel like this would shift more of the score towards frame rate/polling rate. Currently, you have a huge hitwindow, so whether the current frame processes the input or whether the next frame does it doesn't really matter. However, with your suggestion, even 1 frame can change your accuracy by 5%, which doesnt really make sense.

**Reply by smoogipoo** [reply]
Okay, let's try this another way because I still don't understand what you mean.

1. How much should a perfectly-timed (+/- 0ms) "300" give?
2. How much should a perfectly-timed (+/- 21ms @ OD10) "100" give?
3. How much should a poorly-timed (+/- 60ms @ OD10) "100" give?

What I'm assuming you're saying is that (2) and (3) would give different score. Is this correct? If so, as I said, that's not going to happen.

**Reply by Frous-s** [reply]
This change would only affect overall map acc, not score given by the notes based on the timing

**Reply by smoogipoo** [reply]
Accuracy and score are tied together, so changing one changes the other. This could potentially be displayed in the future as an extended statistic similar to the existing ones, but it can't affect score/accuracy.

**Reply by ghost** [reply]
In the mean time, you can use the unstable rate as a similar metric to what you're suggesting

**Comment by abraker95** [comment]
>  a spectrum from 1-80% instead of current accuracy

> basically removing green and yellows being set values of 33.33% and 16.66% and instead be a wide range of values(maybe of 1% precision)

I believe continuous scoring is being proposed here. I already suggested that here: https://github.com/ppy/osu/discussions/15899#discussioncomment-1793966

**Reply by Frous-s** [reply]
Not entirely, but it does fit with the suggestion.

**Reply by abraker95** [reply]
I don't understand how it's not entirely. Acc is tied to score. `acc = score / max_score`. So,

16.66% = 50 score / 300 max score
33.33% = 100 score / 300 max score

Your 1-80% continuous acc requires there to be 30 to 240 continuous score
